x-pack/plugins/ml/public/application/explorer/explorer.js

```diff
@@ -328,6 +328,10 @@ export class ExplorerUI extends React.Component {
                     id="xpack.ml.explorer.topInfuencersTitle"
                     defaultMessage="Top influencers"
                   />
+                  <EuiIconTip
+                    content="This list shows the relative impact of the top influencers on the anomalies in the selected time period. Each influencer has a maximum anomaly score between 0-100 and a total anomaly score for that period."
+                    position="right"
+                  />
```

Comment:
this string requires translation support 
```suggestion
                  <EuiIconTip
                    content={
                      <FormattedMessage
                        id="xpack.ml.explorer.topInfluencersTooltip"
                        defaultMessage="This list shows the relative impact of the top influencers on the anomalies in the selected time period. Each influencer has a maximum anomaly score between 0-100 and a total anomaly score for that period."
                      />
                    }
                    position="right"
                  />
```